README.md

```diff
@@ -28,6 +28,11 @@ composer install -d api
 
 ### Docker
 
+Use the following command to start the environment for the first time:
+```bash
+docker-compose up -d --build
```

Comment:
Is this really needed? As far as I know if started for the first time the containers are automatically built before they are created. Or am I missing something?

Reply:
I'm not sure, we have to test this out.